### PR TITLE
Added purge, sign message and verify message

### PR DESCRIPTION
**Pull request by sbailey-arm**
Missed a few off last PR. This includes everything but copy (as we need to think about copy in the context of Parsec).

Signed-off-by: Samuel Bailey <samuel.bailey@arm.com>

### CODE REVIEW

File: protobuf/psa_purge_key.proto [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
```
Comment (hug-dev) [inline protobuf/psa_purge_key.proto:1]: I think similarly than Copy, Purge has meanings that are not yet clear in Parsec's world so I think it is best to leave it out for now 😃

**Review by hug-dev — COMMENTED** [review]
Thanks for adding the remaining one! I believe that with them we cover all single part crypto operations?

File: protobuf/psa_purge_key.proto (hunk 1 above)
Comment (sbailey-arm) [reply protobuf/psa_purge_key.proto:1]: Roger, will 🧹 it

**Comment by sbailey-arm** [comment]
> 
> 
> Thanks for adding the remaining one! I believe that with them we cover all single part crypto operations?

That is correctamundo. Barring `copy` and `purge`, all other single-part ops here 😄

**Review by hug-dev — APPROVED** [review]

**Review by ionut-arm — APPROVED** [review]

**Pull request merged**